ee/query-service/usage/sqlite/init.go

```diff
@@ -0,0 +1,32 @@
+package sqlite
+
+import (
+	"fmt"
+
+	"github.com/jmoiron/sqlx"
+)
+
+func InitDB(db *sqlx.DB) error {
+	var err error
+	if db == nil {
+		return fmt.Errorf("invalid db connection")
+	}
+
+	table_schema := `CREATE TABLE IF NOT EXISTS usage(
+		id UUID PRIMARY KEY,
+		created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP, 
+		updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+		activation_id UUID, 
```

Comment:
can create a foreign key so invalid activation ids get reported? unless you have the check in the usage prcocessing